atom/browser/atom_browser_main_parts.cc

```diff
@@ -219,9 +205,6 @@ int AtomBrowserMainParts::PreCreateThreads() {
   SecKeychainAddCallback(&KeychainCallback, 0, NULL);
 #endif  // defined(OS_MACOSX)
 
-  device::GeolocationProvider::SetGeolocationDelegate(
-      new AtomGeolocationDelegate());
```

Comment:
is `AtomGeolocationDelegate` still used? If not we should delete.

Reply:
It was removed in the same commit